projects/microphysics/docs/wandb_report_figures/classification-scores.py

```diff
@@ -0,0 +1,209 @@
+import sys
```

Comment:
this is mostly just an ad-hoc script that doesn't make sense to run for every ML model at the moment. If it ends up being re-used then we can refactor it to `workflows/diagnostics`.